app/kubemci/pkg/gcp/healthcheck/healthchecksyncer.go

```diff
@@ -201,13 +204,14 @@ func (h *HealthCheckSyncer) desiredHealthCheck(lbName string, port ingressbe.Ser
 		// Number of healthchecks to fail before the vm is deemed unhealthy.
 		UnhealthyThreshold: DefaultUnhealthyThreshold,
 		Type:               string(port.Protocol),
-		// TODO: Try Kind: compute#healthCheck
+		Kind:               "compute#healthCheck",
 	}
 	switch port.Protocol {
 	case "HTTP":
 		hc.HttpHealthCheck = &compute.HTTPHealthCheck{
 			Port:        port.Port,
 			RequestPath: "/healthz", // TODO(nikhiljindal): Allow customization.
+			ProxyHeader: "NONE",
```

Comment:
I assume we will require this for HTTPs as well? Fine to do it in another PR, since HTTPs doesnt work right now anyway.